config/backup.php

```diff
@@ -235,12 +235,12 @@
             /*
              * If this is an empty string, the name field on the webhook will be used.
              */
-            'username' => env('BACKUP_DISCORD_USERNAME', ''),
+            'username' => env('BACKUP_DISCORD_USERNAME', 'phpVMS Backup'),
 
             /*
              * If this is an empty string, the avatar on the webhook will be used.
              */
-            'avatar_url' => env('BACKUP_DISCORD_AVATAR_URL', ''),
+            'avatar_url' => env('BACKUP_DISCORD_AVATAR_URL', 'https://www.phpvms.net/images/favicon-32x32.png'),
```

Comment:
Does `url('assets/img/favicon.png')` work?

Reply:
I think it works. But will this favicon be less pixelated?